spec/requests/issues_controller_spec.rb

```diff
@@ -149,6 +148,80 @@
         ),
       )
     end
+
+    describe "user with insufficient permissions" do
+      it "does not allow access" do
+        sign_in(user)
+
+        post "/jira/issues.json"
+        expect(response.status).to eq(403)
+      end
+    end
+
+    describe "user in group with permission" do
+      before do
+        SiteSetting.discourse_jira_allowed_groups = Group::AUTO_GROUPS[:moderators]
+        Group.refresh_automatic_groups!
+      end
+
+      it "allows access" do
+        post = Fabricate(:post)
+
+        sign_in(user)
+        mods = Group.find(Group::AUTO_GROUPS[:moderators])
+        mods.add(user)
+
+        stub_request(:post, "https://example.com/rest/api/2/issue").with(
```

Comment:
all of these `stub_request`s  are copy/pasted from the spec on line 64... it's definitely redundant, but I'm not quite sure how to best organize these specs. 🤔